docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -443,8 +532,30 @@ the :ref:`dronecan_fc_tutorial` tutorial.
 
 Arming and Arming Bypass
 ================================
+
+Standard Arming
+*****************
 DroneCAN can use the same advanced arming procedure as all other throttle sources. The details of this arming procedure are covered in the :ref:`manual_advanced_arming` section.
 
+Arming with ArmingStatus
+******************************
+Vertiq modules can arm and disarm based off of DroneCAN's :ref:`ArmingStatus message <arming_status>`. This means that whenever your flight controller broadcasts an ArmingStatus 
+``STATUS_FULLY_ARMED`` message, your module will transition from disarmed to armed, or if already armed, will remain armed. These transitions are not subject to the constraints 
+set by your :ref:`arming throttle region <arming_throttle_regions>`, so any ``STATUS_FULLY_ARMED`` message will arm your module. When your flight controller broadcasts a ``STATUS_DISARMED`` message, your module 
+will similarly transition from armed to disarmed.
```

Comment:
Its sort of mentioned in the hobby thing below, but I think here, early on, you should try to really excessively draw attention to the manual throttle source thing, since you need to set your manual throttle source appropriately to use the throttles. People are often confused by this, so a big bold sentence trying to make it as obvious as possible and pointing to manual throttle source would be good.

Also, now that I think about it, we should probably expose manual arming throttle source as a parameter since that's something people will definitely care about. can you add a task to see what adding that onto our dronecan parameters would be like?